Exploratory_Data_Analysis/Plotting_Systems/lesson

```diff
@@ -102,7 +102,7 @@
   Output: See how the plot changed? No need for you to worry about margins or labels. The package took care of all that for you.
 
 - Class: text
-  Output: Now, for the last plotting system, ggplot2, which is a hybrid of the base and lattice systems. It automatically deals with spacings, text, titles (as Lattice does) but also allows you to annotate by "adding" to a plot (as Base does), so it's the best of both worlds. 
```

Comment:
I think the first comma is unnecessary. "Spacings" doesn't provide any clarification and it's [rarely used](https://books.google.com/ngrams/graph?content=spacing%2Cspacings&year_start=1900&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cspacing%3B%2Cc0%3B.t1%3B%2Cspacings%3B%2Cc0).